source/github-helpers/index.ts

```diff
@@ -22,7 +22,7 @@ Example tag content on public repositories: https://github.com/sindresorhus/refi
 Example tag content on private repositories https://github.com/private/private/commits/master.atom?token=AEAXKWNRHXA2XJ2ZWCMGUUN44LM62
 */
 export const getCurrentBranch = (): string => {
-	return new URL(select<HTMLLinkElement>('[type="application/atom+xml"]')!.href)
+	return new URL(select.last<HTMLLinkElement>('[type="application/atom+xml"]')!.href)
```

Comment:
Where do you see 2 `[type="application/atom+xml"]` elements? This doesn't change the result in my case

Reply:
You saw the gif?

Reply:
I totally glossed over it 🤦‍♂️